assets/questions/bestMeetupLocation.js

```diff
@@ -46,19 +46,27 @@ globalData.questions['bestMeetupLocation'] = {  // eslint-disable-line dot-notat
   },
   tasks: [{
     instructions: [
-      [
-        'N people live in a city with streets that form a regular 2D grid. ',
-        'They start out at different intersections and want to meet up. ',
-        'They are only able to move along the roads (no crossing blocks diagonally etc.). ',
-        'Write a function bestMeetupLocation(people) that returns a meeting place that ',
-        'minimizes the total travel distance.'
-      ].join(''),
-      [
-        'The input is a list of strings indicating the coordinates of the people in a 2D grid. ',
-        'Your function should return a string indicating the best meeting point for N people. ',
-        'For instance, the best meeting point for 3 people at ',
-        '"0, 0", "4, 2" and "2, 4" is "2, 2".'
-      ].join('')
+      {
+        content: [
+          'N people live in a city with streets that form a regular 2D grid. ',
+          'They start out at different intersections and want to meet up. ',
```

Comment:
As a side note, it would be a good idea to add the '[max-len](http://eslint.org/docs/rules/max-len)' rule to our ESLint package. I have raised an [issue #236](https://github.com/google/tie/issues/236) for the same.